public/assets/harvard.js

```diff
@@ -201,3 +201,32 @@ function responsive_search(){
     }
   });
 }
+
+// Keyboard controls for bootstrap dropdown menus
+const toggleDropdown = (event, currentElement) => {
```

Comment:
currentElement argument is not needed, look at the `event.currentTarget` property of jQuery events.  As a bonus, if you drop it and use currentTarget, you can just pass the function directly in the handler assignments!

I'm also not 100% sure we should assume arrow functions are available, but we don't _really_ have a solid supported browsers profile, so that might be me being overly cautious.